dmel_example.py

```diff
@@ -7,18 +7,60 @@
 import stdpopsim
 from stdpopsim import drosophila_melanogaster
 
-chrom = drosophila_melanogaster.genome.chromosomes["chrX"]
+chrom = drosophila_melanogaster.genome.chromosomes["chr2R"]
 recomb_map = chrom.recombination_map()
 
+print("Testing Li and Stephan (2006) model")
+model = drosophila_melanogaster.LiStephanTwoPopulation()
+model.debug()
+
+samples = [msprime.Sample(population=0, time=0),
+           msprime.Sample(population=0, time=0),
+           msprime.Sample(population=1, time=0),
+           msprime.Sample(population=1, time=0)]
+
+
+ts = msprime.simulate(
+    samples=samples,
+#    recombination_map=chrom.recombination_map(),
+    recombination_rate=1e-09,
```

Comment:
This is a placeholder value so that the simulation finishes in a reasonable amount of time, correct? Is there a place we can make a note of that? Maybe just a comment.

Reply:
yes! I can add that comment for sure